build.sh

```diff
@@ -204,18 +228,25 @@ cmake_boolean_option()
 	esac
 }
 
+# ---------------------------------------------------------------------------
+# Let's process the user's wishes
+# ---------------------------------------------------------------------------
+
+features_set_to_autodetect
+parse_args "$@"
+
+if [ $PRINT_HELP -ne 0 ] ; then
+	print_help
+	exit 1
+fi
+
+MAKE_TASKS=$(num_cpu)
```

Comment:
Not sure if I get what you mean in your last comment :-) I've fixed it in master the way I sughested above, but feel free to share additional fixes or suggestions.